_data/projects/horusec.yml

```diff
@@ -0,0 +1,17 @@
+name: Horusec
+desc: Horusec is an open source tool that improves identification of vulnerabilities in your project with just one command.
+site: https://horusec.io
+tags:
+  - java
+  - javascript
+  - security
+  - php
+  - kotlin
+  - typescript
+  - go
+  - pipeline
+  - SAST
+  - CI/CD
```

Comment:
@caroldf07 We need to fix these errors, please change names accordingly.

- Tag 'SAST' contains invalid characters. Allowed characters: a-z, 0-9, +, #, . or -
- Tag 'CI/CD' contains invalid characters. Allowed characters: a-z, 0-9, +, #, . or -